src/Microsoft.DotNet.Darc/src/Darc/Operations/SetRepositoryMergePoliciesOperation.cs

```diff
@@ -124,6 +124,13 @@ public override async Task<int> ExecuteAsync()
                 mergePolicies = initEditorPopUp.MergePolicies;
             }
 
+            IRemote verifyRemote = RemoteFactory.GetRemote(_options, repository, Logger);
+            if (!(await UxHelpers.VerifyMaestroManagedBranchExists(verifyRemote, repository, branch, !_options.Quiet)))
+            {
+                Console.WriteLine("Aborting merge policy creation.");
```

Comment:
NIT: Makes sense adding the reason for aborting to the message?

Reply:
VerifyMaestroManagedBranchExists prints the reason.